content/2017-04-27_mi-pyt.md

```diff
@@ -0,0 +1,183 @@
+
+Součástí hodnocení předmětu byla i semestrální práce. Každý student si mohl vybrat vlastní téma, které navazovalo na nějakou probranou látku. Téma jsme samozřejmě museli schválit. Nevyžadovali jsme, aby studenti svůj kód zveřejnili, ale důrazně jsme to doporučili. Vzniklo tak mnoho nových open-source projektů na GitHubu:
+
+ * [Ovládátko na Raspberry Pi robota](https://github.com/ggljzr/pytRobot)
+ * [Webový dashboard pro Sacred](https://github.com/chovanecm/sacredboard)
+ * [Distribuovaný chat s CLI i GUI](https://github.com/pklejch/distributed-chat)
+ * [Nástroj pro prozkoumávání spektrální analýzy](https://github.com/kozajaku/spectra-analyzer)
+ * [Nástroj pro správu vyvýjených rozšíření pro Chrome a Firefox](https://github.com/melkamar/webstore-manager)
+ * [Implementace známých genetických algortimů](https://github.com/bobirdmi/genetic-algorithms)
+ * [Backend pro webovou aplikaci pro správu turnajů v Ultimate Frisbee](https://github.com/dstlmrk/catcher/tree/mipyt)
+ * [Statistická knihovna pro skrytý Markovův model](https://github.com/lopatovsky/HMMs)
+ * [Sledovač změn na Eduxu](https://github.com/andrenvk/eduxfeed)
+ * [Porovnávač obrázků s GUI](https://github.com/janslifka/imagediffer)
+ * [Zobrazovač výsledků testů z framewroku Cucumber](https://github.com/stengvac/cucumber-sem)
+ * [Síto na GitHub repozitáře k dalšímu zpracování](https://github.com/MarekSuchanek/repocribro)
+
+![Raspberry Pi robot]({filename}/images/robot.jpg)
+Ondřej Červenka semestrálkou [ovládal Raspberry Pi robota](https://github.com/ggljzr/pytRobot) (© 2016 Ondřej Červenka, [MIT License](https://github.com/ggljzr/pytRobot/blob/master/LICENSE))
+
+GitHub, kolem kterého se výuka hodně točila, nás ještě před začátkem semestru zásoboval dostatečným množstvím nálepek a několika 100% slevovými vouchery na trička z [GitHub Shopu](https://github.myshopify.com/). Studentům s nejlepším hodnocením jsme tak po skončení běhu mohli ještě dodatečně udělat radost. Pokud ve výuce používáte GitHub, určitě si také [o swag řekněte](https://education.github.community/t/shopping-bags-get-a-github-swag-bag-for-your-classroom/33).
+
+
+![GitHub nálepky]({filename}/images/github-nalepky.jpg)
+GitHub nálepky
+
+**Tip:** Studenti (a učitelé) mohou zdarma využívat [GitHub Student Developer Pack]. Mimo jiné tak mají k dispozici neomezené množství soukromých repozitářů a nemusíte se tak bát pověstného opisování. My jsme v předmětu využili i službu [Travis CI] pro soukromé repozitáře, která je jinak placená. V balíčku jsou také cloudové služby, na kterých lze provozovat i Python.
+
+[GitHub Student Developer Pack]: https://education.github.com/pack
+
+
+## Zpětná vazba
+
+Po skončení kurzu nám někteří studenti poskytli zpětnou vazbu: buď přes univerzitní anketu, nebo přes formulář, který jsme vystavili na webu. K výsledkům fakultní ankety má přístup jen akademická obec, ale výsledky naší ankety jsou [veřejné][results.csv].
```

Comment:
Ah, tak nic, vypadalo to jako relativní odkaz, ale je to externí odkaz :) už vidím pod odstavcema.